flask_resty/view.py

```diff
@@ -348,12 +348,14 @@ def flush(self):
             # Flushing allows checking invariants without committing.
             self.session.flush()
         except IntegrityError:
+            flask.current_app.logger.exception("Integrity error")
```

Comment:
the stack trace will say `IntegrityError`, so can you make the message here like `"flush failed"`?